frontend/__mocks__/i18next.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-env node */
 import { TFunction } from 'i18next';
 
-const i18next = require.requireActual('i18next');
+const i18next = (jest as any).requireActual('i18next');
```

Comment:
Is there any way to avoid the `as any` throughout? Are we just missing the types for jest or is it another problem?